vendor/github.com/Sirupsen/logrus/logger.go

```diff
@@ -312,6 +312,6 @@ func (logger *Logger) level() Level {
 	return Level(atomic.LoadUint32((*uint32)(&logger.Level)))
 }
 
-func (logger *Logger) setLevel(level Level) {
+func (logger *Logger) SetLevel(level Level) {
```

Comment:
__[golint]__
* [golint] exported method Logger.SetLevel should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->